composer.json

```diff
@@ -31,7 +31,7 @@
         "composer-plugin-api": "^1.0 || ^2.0"
     },
     "require-dev": {
-        "composer/composer": "1.0.* || ^2.0"
+        "composer/composer": "^1.10.23 || ^2.1.9"
```

Comment:
We don't need to worry about the dependent bots. I guess we are good with `^1.0 || ^2.0` .

Reply:
Also this is a dev dependency not a requirement itself. So nothing to worry about.

Reply:
I know that dev-dependencies basicly never impacs the ussage with this project. But github depedabot was complaining, so I fixed it, that this repo is clean and git does not send warnings.